### PR TITLE
RDMA: Add IPsec full-offload support

### PR DESCRIPTION
**Pull request by RoiZilberzwaig**
- Adding support for multiple ports.
- Adding support for multiple devices.

### CODE REVIEW

File: common.sh [hunk 1]
```diff
@@ -153,30 +153,32 @@ get_ips() {
...
-            fatal "Interface ${CLIENT_NETDEVS[${#CLIENT_NETDEVS[@]}-1]} on ${CLIENT_TRUSTED} seems not to have an IPv4."
+            fatal "Interface ${CLIENT_NETDEVS[i]} on ${CLIENT_TRUSTED} seems not to have an IPv4."
         fi
         CLIENT_IPS+=("$( echo "$ip_str" | grep -ioP '(?<=inet )\d+\.\d+\.\d+\.\d+' | xargs | tr ' ' ',')")
         CLIENT_IPS_MASK+=("$( echo "$ip_str" | grep -ioP "(?<=${CLIENT_IPS[i]}/)\d+" | xargs | tr ' ' ',')")
         [ -z "${CLIENT_IPS[${#CLIENT_IPS[@]}-1]}" ] &&
-            fatal "Can't find a client IP associated with the net device '${CLIENT_NETDEVS[${#CLIENT_NETDEVS[@]}-1]}'." ||
-            log "INFO: Found $(awk -F',' '{print NF}' <<<"${CLIENT_IPS[${#CLIENT_IPS[@]}-1]}") IPs associated with the client net device '${CLIENT_NETDEVS[${#CLIENT_NETDEVS[@]}-1]}'."
+        fatal "Can't find a client IP associated with the net device '${CLIENT_NETDEVS[i]}'." ||
+        log "INFO: Found $(awk -F',' '{print NF}' <<<"${CLIENT_IPS[${#CLIENT_IPS[@]}-1]}") IPs associated with the client net device '${CLIENT_NETDEVS[i]}'."
```
Comment (blochl) [inline common.sh:180]: The two lines above should be indented (please see the 'before').

File: ngc_rdma_test.sh [hunk 2]
```diff
@@ -65,6 +67,14 @@ do
             RDMA_UNIDIR=true
             shift
             ;;
+        --mtu=*)
+            MTU_SIZE="${1#*=}"
+            shift
+            ;;
```
Comment (blochl) [inline ngc_rdma_test.sh:73]: Do you add MTU here as well?

File: common.sh (hunk 1 above)
Comment (RoiZilberzwaig) [reply common.sh]: Fixed

File: ngc_rdma_test.sh (hunk 2 above)
Comment (RoiZilberzwaig) [reply ngc_rdma_test.sh]: Fixed

**Review by blochl — APPROVED** [review]
Thanks!

**Pull request merged**